src/utilities/valueFromAST.js

```diff
@@ -57,13 +59,17 @@ export function valueFromAST(
   }
 
   if (!valueAST) {
+    return undefined;
+  }
+
+  if (valueAST.kind === Kind.NULL) {
```

Comment:
I think you should include a short comment explaining that this is intentionally returning the *value* `null`, which is different from simply `return;`